deploy/operator.yaml

```diff
@@ -16,7 +16,7 @@ spec:
       containers:
         - name: keycloak-operator
           # Replace this with the built image name
-          image: quay.io/keycloak/keycloak-operator:master
+          image: quay.io/keycloak/keycloak-operator:11.0.0
```

Comment:
@slaskawi question. Why not `latest`?

Reply:
I think this should be `master`

Reply:
What's the difference between `master` and `latest`?
In any case, it shouldn't be `11.0.0`.

Reply:
Since #238 was merged, `master` will always be an image based on the head commit of the master branch. `latest` will not be updated automatically once the autobuild in Quay is disabled (I think?).

I'd suggest `latest` be aliased to the the most recent version tag.